tests/perf/actor_id_scale/test.js

```diff
@@ -0,0 +1,55 @@
+import http from "k6/http";
```

Comment:
I don't see a package.json. Or k6 doesn't need it?

Reply:
The k6 javascript runtime is backed by https://github.com/dop251/goja,

Since this script is not aimed to be backed by a nodejs runtime or a headless browser, you can't use functions like from `window` `os`, `fs`, etc. When you need to include libraries/dependencies you have to bundle it all together using a bundling tool like webpack or something similar.

The imports that I'm using are all [provided by k6 in runtime](https://k6.io/docs/javascript-api/), that's why I don't need any node_modules nor package.json, because I don't depend on external libraries. btw you can also use remote module import https://k6.io/docs/using-k6/modules/#the-jslib-repository

Reply:
Perfect, thanks for explaining!